django_orm_profiler/core/emitter.py

```diff
@@ -23,6 +25,6 @@ def __init__(self, host, port):
     """
     def emit(self, payload):
         self.emit_socket.sendto(
-            json.dumps(payload),
+            DjangoJSONEncoder().encode(payload).encode(),
```

Comment:
@bragradon thanks for the upgrades. What is the behavior when encoding a string with `encode()` like you have above without passing an encoding type? Wondering if we should be explicit here. Happy to merge this in after we this bit out.

Reply:
The `encode()` without parameters will default to UTF-8. But I agree having it explicit is better. I've updated the PR

Reply:
The `encode()` without parameters will default to UTF-8. I agree with making it explicit so have updated the PR